torch/csrc/distributed/c10d/ProcessGroupNCCL.cpp

```diff
@@ -4263,7 +4264,7 @@ c10::intrusive_ptr<Work> ProcessGroupNCCL::send(
 
   RECORD_PARAM_COMMS_DATA(
       static_cast<int>(
-          this->getSequenceNumberForGroup() + 1), // seq + 1 to match collective
+          this->seqP2P_ + 1), // seqP2P_ + 1 to match pointToPoint op
```

Comment:
seems better to use tuple `(seq, is_p2p)` for `RECORD_PARAM_COMMS_DATA`. Otherewise, replayer needs to copy the logic of PyTorch to check `collectiveName_`. Keep consistant pattern would be better.